src/main/java/team/chisel/Chisel.java

```diff
@@ -138,6 +139,9 @@ public void preInit(FMLPreInitializationEvent event) {
 		if (Loader.isModLoaded("ForgeMultipart")) {
 			new FMPCompat().init();
 		}
+		if (Loader.isModLoaded("dreamcraft")) {
+		    gtnhLoaded = true;
```

Comment:
We need to detect GTNH before `Features.preInit()`, otherwise unlocalized name remains non-GTNH version

Reply:
Can you elaborate that a bit? I don't know what you mean...

Reply:
Unlocalized names are set in `ItemChisel` constructor. They are instantiated in `Features.CHISEL#addItems`, which is called by `Features#loadItems`. As a result, unlocalized names are set before we set `gtnhLoaded = true`.

Reply:
Ah, good catch